extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1088,6 +1090,9 @@ export class AmpA4A extends AMP.BaseElement {
           // modified url.
           'src': xhrFor(this.win).getCorsUrl(this.win, adUrl),
         }, SHARED_IFRAME_PROPERTIES));
+    const sentinel = generateSentinel(window);
+    const metadata = getContextMetadata(window, iframe, sentinel);
+    iframe.setAttribute('name', encodeURIComponent(JSON.stringify(metadata)));
```

Comment:
My concern is an XSS injection, because we're writing this data into the page that could contain in the JSON a bunch of stuff that we're not controlling. So I meant that they could close the quote on the name attribute, and then like open a script tag or something. Like if the json object was:

`{"foo":"<script>alert("foo");</script>}`

that would be written right in the page.